packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/add.constants.js

```diff
@@ -1,7 +1,8 @@
 export const PRESET_IMAGE = 'preset-image';
 export const BUILD_IMAGE = 'build-image';
-export const EXTENSION = ['.h5', '.onnx', '.pmml'];
+export const EXTENSION = ['.h5', '.onnx', '.pmml', '.pt', '.pth', '.pkl'];
 export const FOLDER_MODE = 'folder';
 export const FILE_MODE = 'file';
-export const COMPATIBILITY_GUIDE_URL =
-  'https://docs.ovh.com/gb/en/serving-engine/compatibility-matrix';
+export const COMPATIBILITY_GUIDE_URL = {
+  en_GB: 'https://docs.ovh.com/gb/en/ml-serving/compatibility-matrix',
```

Comment:
what about guide links for other languages? fallback to English if not available.

Reply:
I do fallback to English:
```
href="{{ $ctrl.COMPATIBILITY_GUIDE_URL[$ctrl.user.language] || $ctrl.COMPATIBILITY_GUIDE_URL['en_GB'] }}"
```